src/wp-admin/install.php

```diff
@@ -315,7 +315,7 @@ function display_setup_form( $error = null ) {
 		display_header();
 ?>
 <h1><?php _ex( 'Welcome', 'Hello' ); ?></h1>
-<p><?php _e( 'Welcome to the famous five-minute ClassicPress installation process! Just fill in the information below and you&#8217;ll be on your way to using the most extendable and powerful personal publishing platform in the world.' ); ?></p>
+<p><?php _e( 'Welcome to the ClassicPress installation process! ClassicPress brings all of the latest features the best modern CMS while keeping the old and proven way to compose and manage your content. Just fill in the information below and you&#8217;ll be on your way to using the most extendable and powerful personal publishing platform in the world.' ); ?></p>
```

Comment:
It seems there is a word missing in:

"ClassicPress brings all of the latest features the best modern CMS while keeping..."